docs/crd/CRDS.adoc

```diff
@@ -28,9 +28,9 @@ link:maistra.io_ServiceMeshExtension_v1alpha1.adoc[ServiceMeshExtension]:: Servi
 
 === v1
 
-link:federation.maistra.io_ExportedServiceSet_v1.adoc[ExportedServiceSet]:: ExportedServiceSet is the Schema for configuring exported services.  The name of the ExportedServiceSet resource must match the name of a ServiceMeshPeer resource defining the remote mesh to which the services will be exported.
+link:federation.maistra.io_ExportedServiceSet_v1.adoc[ExportedServiceSet]:: ExportedServiceSet is the Schema for configuring exported services. It must be created in the same namespace as the control plane. The name of the ExportedServiceSet resource must match the name of a ServiceMeshPeer resource defining the remote mesh to which the services will be exported. This implies there will be at most one ExportedServiceSet resource per control plane.
```

Comment:
The last sentence is not correct. There is at most one ExportedServiceSet resource per peer. 

For example: When you have three meshes (red, green, blue), the red mesh will have two ServiceMeshPeer resources (green-mesh and blue-mesh) and two ExportedServiceSet resources (green-mesh and blue-mesh).

Reply:
Updated